cfgov/tccp/jinja2/tccp/card.html

```diff
@@ -690,6 +690,37 @@ <h2>Rewards and perks</h2>
             </dd>
         {% endif %}
     </dl>
+    <h2>Contact information</h2>
+    <dl class="m-list m-list__card-details">
+        {% if card.website_for_consumer %}
+            <dt>Website</dt>
+            <dd class="a-card-url">
+                {# Some issuers submitted more than one URL for a card. In all
+                    of those instances, the URLs are separated with a space, so
+                    we'll only turn the submitted URL into a link if it doesn't
+                    have a space in it for now. TODO: maybe split multiple URLs
+                    up.
+                #}
+                {% if ' ' is not in card.website_for_consumer %}
+                    {# TODO: Do we need to run this through link middelware? #}
+                    <a href="{{ card.website_for_consumer }}">
+                        {{ card.website_for_consumer }}
+                    </a>
+                {% else %}
+                    {{ card.website_for_consumer }}
+                {% endif %}
+            </dd>
+        {% endif %}
+        {% if card.telephone_number_for_consumers %}
+            <dt>Phone</dt>
+            <dd>
+                {# TODO: Do we want this marked up as a phone link? #}
+                <a href="tel:{{ card.telephone_number_for_consumers }}">
```

Comment:
According to [the RFC for tel: links](https://www.rfc-editor.org/rfc/rfc3966):

> "tel" URIs MUST NOT use spaces in visual separators to avoid excessive escaping

This means that values like `(800) 555-1234` are technically invalid.

Similarly,

> In some countries, it is common to write phone numbers with alphabetic characters corresponding to certain numbers on the telephone keypad.  The URI format does not support this notation, as the mapping from alphabetic characters to digits is not completely uniform internationally

So values like `(800) BUY-CARD` are also invalid.

Both of these can exist in the data. I'm not sure if this means we should avoid using `tel:` links (and it's possible that Safari/Chrome/etc. support these just fine in the US) but maybe something to test?

Reply:
I removed the `tel` links in c087f88 and now display just plain text phone numbers